generated/proto/policypb/policy.proto

```diff
@@ -44,3 +44,9 @@ message Policy {
   StoragePolicy storage_policy = 1;
   repeated aggregationpb.AggregationType aggregation_types = 2;
 }
+
+enum DropPolicy {
+  NONE = 0;
```

Comment:
Since it's an optional field, the default value 0 is going to be set on existing rules.  I want that to mean "no drop" and not "unknown" which will cause the code to log errors, etc.